http/oidc/src/main/java/org/wildfly/security/http/oidc/Oidc.java

```diff
@@ -180,7 +180,14 @@ public class Oidc {
     public static final String CONFIDENTIAL_PORT = "confidential-port";
     public static final String ENABLE_BASIC_AUTH = "enable-basic-auth";
     public static final String PROVIDER_URL = "provider-url";
-
+    public static final String LOGOUT_PATH = "logout-path";
+    public static final String LOGOUT_CALLBACK_PATH = "logout-callback-path";
+    public static final String POST_LOGOUT_REDIRECT_URI= "post-logout-redirect-uri";
+    public static final String LOGOUT_SESSION_REQUIRED = "logout-session-required";
+    public static final String logoutPath = "/logout";
+    public static final String logoutCallbackPath = "/logout/callback";
+    public static final int backChannelLogoutSessionInvalidationLimit = 16;
```

Comment:
We should use upper case for the above three variable names since these are constants. We can also prefix with DEFAULT, e.g., DEFAULT_LOGOUT_PATH="/logout"

Reply:
done